### ISSUE TITLE
What's problem with my allDay setting?

### ISSUE THREAD

**Issue by DivingPigy** [CroudTech/vue-fullcalendar#145]
I have ser allDay configuration in json, but it didn't work. Any criticle point I miss ?

code:
`allDay: true,`

Result:

![default](https://user-images.githubusercontent.com/26334368/42929335-58637420-8b6c-11e8-9891-0ef053e8d46f.png)

**Comment by DivingPigy** [comment]
I read the the sample code on sandbox, the host set the attributte allDay to be true. Then it displayed in the top allDay section.
Also, I read in officical docs to do the same thing. Howere, not working for me.
![default](https://user-images.githubusercontent.com/26334368/42929709-69eb9f1e-8b6d-11e8-90c3-3f019d19f9f5.png)

**Comment by DivingPigy** [comment]
The birthday event just fills the vetical day cell. Not friendly for user.

**Comment by BrockReece** [comment]
Can you send me a sample of your event data?

**Comment by DivingPigy** [comment]
`if(res.data.data[i].type == "3"){
                        
                        event_array.push({
                        title: res.data.data[i].title,
                        desc: res.data.data[i].desc,
                        start: date_arr[0] + "-" + res.data.data[i].start_time,
                                          
                        id: res.data.data[i].id,
                        color: 'red',
                        allDay: true,
                      });
                    }`

**Comment by DivingPigy** [comment]
I set allDay true here.
The events are added to calendar by json feed.

**Comment by DivingPigy** [comment]
I have done this before. Let me try again.

**Comment by DivingPigy** [comment]
`                        start: date_arr[0] + "-" + res.data.data[i].start_time + " 10:00:00",
                        end: res.data.data[i].end_time + " 11:00:00",`

It doesn't work.

**Comment by BrockReece** [comment]
What does your base fullcalendar config look like?

**Comment by BrockReece** [comment]
Which version of vue-full-calendar are you using? I will try and replicate the issue.

**Comment by DivingPigy** [comment]
I fountd that in the file: node_modules/vue-full-calendar/package.json -> "_id": "vue-full-calendar@4.0.0-1",

Is this help , Sir?

**Comment by BrockReece** [comment]
Try downgrading to 2.x
4.x is in prerelease at the moment.

**Comment by DivingPigy** [comment]
I modify the package.json file, change the item:
1)  "vue-full-calendar": "^4.0.0-1"   ==>>   "vue-full-calendar": "^2.0.0-1"
2) Run in the console: npm install  npm run dev
But it didn't work. Any mistake I make?

**Comment by DivingPigy** [comment]
A new try:
    package.json file,change the item:
        1)  "vue-full-calendar": "^2.7.0-0",
        2)  Run in the console: npm install npm run dev

Not work.

**Comment by DivingPigy** [comment]
A new try on v4.0.0-1, not work.

**Comment by BrockReece** [comment]
try ^2.6.1

**Comment by DivingPigy** [comment]
Now, the events are displayed in the allDay section. However, I click the month-nav to change to month-view, the events displayed in allDay will dispear.

For example: this is my click sequence: week(events exist in allDay section) => month(events dispear) => week(dispear)

**Comment by DivingPigy** [comment]
So wired, the allDay section doesn't work again.
